transformer_lens/utils.py

```diff
@@ -24,7 +24,7 @@
 from jaxtyping import Float, Int
 
 
-def _select_compatible_kwargs(
+def select_compatible_kwargs(
```

Comment:
I don't think this change is needed? I think using `_` on functions only used in a given file is a good idea - I just don't think it is worth breaking backwards compatibility for

Reply:
Ok I can revert that, although I coded this function a few days ago so it's probably not a worry for backward compatibility. I'd like a quick review from @jbloomAus just to be sure we all agree.

Reply:
The docs are on Neel Nanda's website (https://neelnanda-io.github.io/TransformerLens/), so I don't think I can regenerate it. When I do `poetry run sphinx-autobuild -b dirhtml ./source build/html` it just generates a local server on my computer, that no one else can access. I'm not sure but I guess Neel updates it regularly and manually.

Reply:
And our Github actions should automatically push this to Neel's website : )